### ISSUE TITLE
[8.0][BUG] Marketing Menu access right

### ISSUE THREAD

**Issue by Odew** [odoo/odoo#12136]
Impacted versions: 8.0

Steps to reproduce: 
- on runbot with database with all modules installed.
- create a new user with only the 'Human Resources/Employee' group access right
- connect with this new created user.
- this user can see the "Marketing" menu, create a new mass mailing, ...

Current behavior:
- a user with no marketing access right can see the "Marketing" menu and do everything with the marketing stuff.

Expected behavior:
- only "Marketing/User" group members can see the "Marketing" menu and use it.

Video/Screenshot link (optional):
![user_creation](https://cloud.githubusercontent.com/assets/15997630/15472076/91292b68-20f9-11e6-8ca2-fd04f4863c97.png)
![user_test_view](https://cloud.githubusercontent.com/assets/15997630/15472078/92d88dd2-20f9-11e6-8317-584ae9109046.png)

**Comment by Yenthe666** [comment]
Hi @Odew,

Thank you for creating an issue (that is also nicely explained)!
I can confirm this issue and the fact that you get full access to marketing with only HR rights.

@mart-e please assign somebody to this?

Regards,
Yenthe

**Comment by mart-e** [comment]
Hello,

We have made improvements in the default groups for 9.0 (for instance at 6375a4bcf4041f898830762ac892c75ab689ce15 ) but the point is that the access to the model mass mailing is quite open 8.0. However it is not something we can change in stable version (may break flow of existing users).

The security rules are default values that may be customized by the users, it depends a lot of the organisation structure and we keep it usually quite permissive. In this case it is probably not optimal but we do not consider this as a bug but rather as an improvement. Closing as the improvement has already been done in 9.0
